db/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM mariadb:10.2
+
+ENV MYSQL_USER=beaker \
```

Comment:
Allow user to define username/password as pass it everywhere as a variable?

Reply:
We can also do this but for a development environment I don't think it would be neither necessary nor useful.